dev-server/documents/html/injectable-frame.mustache

```diff
@@ -0,0 +1,1543 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>The Disappearance of Lady Carfax, by Arthur Conan Doyle</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
+    <script>
+      window.hypothesisConfig = function () {
+        return {
+          "subFrame": true
```

Comment:
There aren't any references to `subFrame` elsewhere in the code. I think you intended to use `subFrameIdentifier` here? See `annotator/config/index.js` and `annotator/index.js`.

Reply:
This is a residue of me playing with configuration, but it is not necessary. I will remove it.